storybook/src/pages/amsterdam/NavigationPage/NavigationPage.stories.tsx

```diff
@@ -0,0 +1,248 @@
+      <Grid gapVertical="large" paddingBottom="large">
+        <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }}>
+          <PageHeading className="ams-mb-m">Burgerzaken</PageHeading>
+          <Paragraph size="large">
+            Heeft u een paspoort, rijbewijs of uittreksel nodig? Of wilt u een verhuizing doorgeven of een geboorte
+            aangeven? Op deze pagina vindt u alle informatie en regelzaken rondom Burgerzaken.
+          </Paragraph>
+        </Grid.Cell>
+        <LinkGroupGridCells linkGroups={burgerzakenLinks} />
+      </Grid>
+    ),
+  },
+}
+
+const parkerenLorem =
+  'Aenean lorem elit, commodo eget elit sit amet, auctor venenatis ligula. Phasellus faucibus fringilla viverra. Aliquam vestibulum vehicula enim, ac consequat nisl mattis id.'
+
+export const WithMap: StoryObj = {
+  args: {
+    children: (
+      <>
+        <Grid gapVertical="large" paddingBottom="large">
+          <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }}>
+            <PageHeading className="ams-mb-m">Parkeren</PageHeading>
+            <Paragraph className="ams-mb-m" size="large">
+              {parkerenLorem}
+            </Paragraph>
+            <Paragraph className="ams-mb-m">{parkerenLorem}</Paragraph>
+            <Paragraph>{parkerenLorem}</Paragraph>
+          </Grid.Cell>
+          <LinkGroupGridCells linkGroups={parkerenLinks}></LinkGroupGridCells>
+        </Grid>
+        <Spotlight>
+          <Grid paddingVertical="large">
+            <Grid.Cell span={{ narrow: 4, medium: 4, wide: 5 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
+              <Heading className="ams-mb-m" color="inverse" level={2}>
+                Parkeertarieven
+              </Heading>
+              <Paragraph className="ams-mb-m" color="inverse">
+                Voor Weesp kijkt u op{' '}
+                <Link color="inverse" href="#" variant="inline">
+                  Parkeertarieven stadsgebied Weesp
+                </Link>
+                .
+              </Paragraph>
+              <UnorderedList className="ams-mb-l" color="inverse">
+                <UnorderedList.Item>Vul een adres in om de mogelijkheden te bekijken</UnorderedList.Item>
+                <UnorderedList.Item>
+                  Als u op een feestdag wilt parkeren, bekijk dan ook{' '}
+                  <Link color="inverse" href="#" variant="inline">
+                    Parkeren op feestdagen
+                  </Link>
+                  .
+                </UnorderedList.Item>
+              </UnorderedList>
+              <SearchField>
+                <SearchField.Input />
+                <SearchField.Button />
+              </SearchField>
+            </Grid.Cell>
+          </Grid>
+        </Spotlight>
+        <Image
+          alt=""
+          aspectRatio="16:9"
+          className="ams-mb-xl"
+          key={1}
+          src="http://www.parkerenindestad.nl/images/amsterdam/parkeren-in-amsterdam.jpg"
+          style={{ inlineSize: '100%' }}
```

Comment:
We could demonstrate how to use CSS modules in such cases. I've noticed allowing inline styles opens the door to a variety of bad CSS. Flex on Grid elements etc.